pkgs/applications/networking/remote/xrdp/default.nix

```diff
@@ -34,7 +34,7 @@ let
   };
 
   xrdp = stdenv.mkDerivation rec {
-    version = "0.9.21.1";
+    version = "0.9.22";
     pname = "xrdp";
 
     src = fetchFromGitHub {
```

Comment:
No hash change?

Reply:
Oh my, that's why I shouldn't do this stuff right after getting out of bed. Fixed now.